backend/db/refresh.py

```diff
@@ -57,6 +59,7 @@ def refresh_db(
         if since and dp.parse(since) > dp.parse(last_refresh) and not force_non_contiguity:
             raise ValueError(SINCE_ERR)
         since = since if since else last_refresh
+        since = dp.parse(since) - timedelta(hours=buffer_hours)
```

Comment:
### 2 day buffer
I've set a buffer of 48 hours now. I agree it makes sense for the buffer to be longer for the cronjob, but with my limited time, it's easier/fine to do it this way for now. And I'll be checking every 2 weeks as well to make sure things are working as expected.